extension/index.lambda.js

```diff
@@ -22,6 +35,15 @@ exports.handler = async (event) => {
     }
   } catch (e) {
     logger.error(e, `Unexpected error when processing event`)
-    throw e
+    const errorObj = {
```

Comment:
Would be nice to use the same pattern for an error message like you're doing for index.googleFunction, as those error handlers duplicate in a different part of the module (payment.controller and index.lambda), would be nice to find a place same parts of it a function, you might return the result, for index.lambda you might merge/extend `responseType: 'FailedValidation',` inside the request. What do you think?

````js
} catch (err) {
    const errorMessage = `Unexpected error (Payment ID: ${paymentObj?.id}): ${err.message}. `
    const errorStackTrace = `Unexpected error (Payment ID: ${
      paymentObj?.id
    }): ${JSON.stringify(serializeError(err))}`
    logger.error(errorStackTrace)

    return {
      responseType: 'FailedValidation',
      errors: [
        {
          code: 'General',
          message: errorMessage,
        },
      ],
    }
  }
````

Reply:
Yup. You're right. It can be refactored.
https://github.com/commercetools/commercetools-adyen-integration/pull/630/commits/b219a065378b95602f19a09bfd9beee3c59a6ec7